install/kubernetes/istio-gateway.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: istio-gateway
+spec:
+  selector:
+    istio: ingressgateway # use istio default controller
+  servers:
+  - port:
+      number: 15007
+      name: pilot-http-discovery
+      protocol: TCP
+    hosts:
+    - "*"
+  - port:
+      number: 15010
+      name: pilot-grpc-xds
+      protocol: TCP
+    hosts:
+    - "*"
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: pilot-http-discovery
+spec:
+  hosts:
+  - "*"
```

Comment:
Define two `VirtualService` with hosts as `*` will not work, you may want to merge the two `VirtualService` to one.

Reply:
well, it does work as I indicated in the description. The hosts name doesn't seem to matter in the case of tcp (non http) ports.

Reply:
If you create two virtualservices with host as `*`, pilot will report error, please refer to https://github.com/istio/istio/issues/6045 for detail